.gitignore

```
@@ -3,3 +3,4 @@ db/*.rdb
 usage_log.txt
 .DS_Store
 dist-lite/
+local*
```

```suggestion
local.*
```
Avoid accidentally including any filename beginning with "local" in the future